File: src/main/scala/MyStrategy.scala
import model.{Car, Game, Move, World}

final class MyStrategy extends Strategy {
  override def move(self: Car, world: World, game: Game, move: Move) {
    move.enginePower = 1.0D
    move.throwProjectile = true
    move.spillOil = true

    if (world.tick > game.initialFreezeDurationTicks) {
      move.useNitro = true
    }
  }
}

File: src/main/scala/MyStrategy.java
import model.Car;
import model.Game;
import model.Move;
import model.World;

public final class MyStrategy implements Strategy {
    @Override
    public void move(Car self, World world, Game game, Move move) {
        move.setEnginePower(1.0D);
        move.setThrowProjectile(true);
        move.setSpillOil(true);

        if (world.getTick() > game.getInitialFreezeDurationTicks()) {
            move.setUseNitro(true);
        }
    }
}
